w3-clock.md

```diff
@@ -0,0 +1,82 @@
+# Merkle Clock
+
+![wip](https://img.shields.io/badge/status-wip-orange.svg?style=flat-square)
+
+## Editors
+
+* [Alan Shaw](https://github.com/alanshaw), [DAG House](https://dag.house/)
+
+## Authors
+
+* [Alan Shaw](https://github.com/alanshaw), [DAG House](https://dag.house/)
+
+## Language
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in [RFC 2119](https://datatracker.ietf.org/doc/html/rfc2119).
+
+## Abstract
+
+This specification describes a method of managing a [Merkle Clock](https://arxiv.org/pdf/2004.00107.pdf) via UCAN invocations. Specifically, we cover two capabilities:
+
+1. Advancing a clock by adding a new event.
+2. Fetching the current head of the clock (the most recent event(s) as known to the clock).
+
+## Data Format
+
+The data format of a Merkle Clock event:
+
+IPLD Schema
+
+```ipldsch
+type Event struct {
+  parents [&Event]
+  data &Data
+}
+
+type Data = Any
+```
+
+## Advancing a clock
+
+An agent may invoke a `clock/advance` capability to advance the head of a merkle clock.
```

Comment:
what if we call it `clock/tick`?

Reply:
I don't have _super_ strong feelings but IMO "tick" _suggests_ a bit more that this is a traditional "global time" based clock that will tick periodically, however merkle clocks are logical clocks that encode causal information - time only advances for them when new events are added, hence "advance" being the verb here.